src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java

```diff
@@ -9,4 +9,5 @@
 @Repository
 public interface UserRepository extends CrudRepository<User, Integer> {
     Optional<User> findByUsername(String username);
+    Optional<User> deleteByUsername(String username);
```

Comment:
Do we need this method?

Reply:
I don't think so and I think you already removed it - I'll make sure :)